### ISSUE TITLE
Add Cors middleware to the API

### ISSUE THREAD

**Issue by saitodeveloper** [saitodeveloper/vestibular-api#51]
As a web browser end-user, I need to request resources from the API using web browsers validating the Cors-based mechanism.

## Description
Request multiple APIs is blocked by default for almost all browsers. It is necessary to add the Cors validation on the backend side.

## Definition of done
- [x] Create a middleware to validate the Cors-based mechanism.
- [x] Frontend must be able to send cookies with the domain for development and using the SameSite Lax.

## Out of scope
- Use the cors to allow localhost and official DNS name. We don't have the name yet.
- Unit and integration tests.
